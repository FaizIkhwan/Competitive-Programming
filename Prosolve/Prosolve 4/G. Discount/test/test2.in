5
33
434
5199
59300
99999