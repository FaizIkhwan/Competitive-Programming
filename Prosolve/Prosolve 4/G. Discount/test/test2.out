RM 33.00
RM 434.00
RM 4419.15
RM 50405.00
RM 84999.15
